dot/emacs

```diff
@@ -12,3 +12,47 @@
 (require 'bivio-init)
 (ignore-errors
     (require 'post-bivio-init))
+(custom-set-variables
```

Comment:
Are you sure you want these changes? They are automated. What prompted them?

Reply:
Fix is in https://github.com/biviosoftware/home-env/pull/64 please review and merge. Good catch